src/AttributeTranslatorInterface.php

```diff
@@ -5,11 +5,13 @@
 namespace Yiisoft\Validator;
```

Comment:
A `TranslatorAttributeTranslator` marked as completed in #358, however constructor and method `translate()` are not documented.